src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetGeneric.java

```diff
@@ -127,11 +128,10 @@ public boolean canUse(ItemStack tool, EntityPlayer player) {
     }
 
     public void applyDamage(ItemStack tool, EntityPlayer player) {
-        if(tool.hasCapability(CapabilityEnergy.ENERGY, null)) {
+        if (tool.hasCapability(CapabilityEnergy.ENERGY, null)) {
             IEnergyStorage energy = CapabilityProviderEnergy.getCap(tool);
             energy.extractEnergy(getEnergyCost(tool), false);
-        }
-        else
+        } else
```

Comment:
no

Reply:
For some reason it looks normal on my computer..
```java
public void applyDamage(ItemStack tool, EntityPlayer player) {
    if (tool.hasCapability(CapabilityEnergy.ENERGY, null)) {
        IEnergyStorage energy = CapabilityProviderEnergy.getCap(tool);
        energy.extractEnergy(getEnergyCost(tool), false);
    } else
        tool.damageItem(getDamageCost(tool), player);
}
```

Reply:
Are we talking about 
```java
        }
        else
```
is not ok or
```java
        } else
```
is not ok?

Reply:
It should either be
```
if (b)
    foo();
else
    bar();
```
or
```
if (b) {
    foo();
    bar();
} else {
    baz();
    bah();
}
```
or
```
if (b) {
    foo();
    bar();
} else
    baz();
```
or
```
if (b)
    foo();
else {
    bar();
    baz();
}
```

Reply:
Ok then, I think @MichaelHillcox misread the changes, I fixed the fact that `else` is on a new line. :)